From 736195caa86c1bdd9470529a0c22674a60145e4b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Adam=20=C5=A0tevko?= <adam.stevko@gmail.com>
Date: Tue, 3 May 2016 10:20:20 +0200
Subject: [PATCH] OS-5832 disks.*.zfs_recsize attribute does not exist Reviewed
 by: Robert Mustacchi <rm@joyent.com> Approved by: Robert Mustacchi
 <rm@joyent.com>

---
 src/vm/man/vmadm.1m.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/vm/man/vmadm.1m.md b/src/vm/man/vmadm.1m.md
index f780d1b6..80e69174 100644
--- a/src/vm/man/vmadm.1m.md
+++ b/src/vm/man/vmadm.1m.md
@@ -2060,7 +2060,7 @@ tab-complete UUIDs rather than having to type them out for every command.
         changing the file system's recordsize affects only files created
         after the setting is changed; existing files are unaffected.
 
-        NOTE: to change this property for KVM, see disks.*.zfs_recsize above.
+        NOTE: to change this property for KVM, see disks.*.block_size above.
 
         type: integer (record size in bytes, 512 to 131072, must be power of 2)
         vmtype: OS
-- 
2.21.0

